src/tiled/changemapobject.cpp

```diff
@@ -204,6 +204,20 @@ void DetachObjects::undo()
     emit mMapDocument->mapObjectModel()->objectsChanged(mMapObjects);
 }
 
+void copyProperties(const MapObject *source, MapObject *target) {
+    target->setName(source->name());
+    target->setSize(source->size());
+    target->setType(source->type());
+    target->setTextData(source->textData());
+    target->setPolygon(source->polygon());
+    target->setShape(source->shape());
+    target->setCell(source->cell());
+    target->setRotation(source->rotation());
+    target->setVisible(source->VisibleProperty);
```

Comment:
I just noticed this little issue. I won't change it now on master to avoid conflicts, so please fix this to `source->isVisible()` as part of this PR.